packages/react-dom/src/__tests__/ReactDOMServerPartialHydration-test.internal.js

```diff
@@ -3435,9 +3433,6 @@ describe('ReactDOMServerPartialHydration', () => {
       {withoutStack: 1},
     );
     expect(Scheduler).toHaveYielded([
-      'Log recoverable error: Hydration failed because the initial UI does not match what was rendered on the server.',
```

Comment:
It's interesting that this changed. What happened before is the second suspense boundary would attempt to hydrate the `<div>` from the prior step which failed b/c it was a `<span>`. Hydration at this point is sort of pointless but I understand the reason is to allow these functions to warm up even though they will end up client rendering.

B/c of the changes in top level contexts (head, body, or the root of the app) the suspense boundary now skips over the <div> (assuming it was inserted by some 3rd party) and finds the expected Suspense boundary that follows. This no longer leads to a second hydration error